app/src/main/java/com/kanshu/kanshu/ArticleActivity.java

```diff
@@ -34,7 +32,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_article);
+        setContentView(R.layout.activity_main);
```

Comment:
main doesn't seem right for something that is reused. Can't think of a better name though. activity_with_nav_drawer_and_tabs seems too long.